addons/web/static/src/views/model.js

```diff
@@ -145,6 +145,9 @@ export function useModel(ModelClass, params, options = {}) {
             await options.onWillStart();
         }
         await load(component.props);
+        if (options.onWillStartAfterLoad) {
```

Comment:
not really sure about that... seems weird. again, can't you just subclass a model and hook your code at the end of `model.load` ?

Reply:
The bank reco widget has 2 views: kanban on the left, form on the right
If we add a listener on the model, the view is opened in a really weird way because the kanban is loaded and rendered leaving the right-side blank. Then, the form is initialize and displayed on the right.